Fix tests as reported by ${WRKSRC}/_build/meson-logs/testlog.txt

# Contents don't match expected contents
# --- /tmp/.NT7X02	2025-01-22 06:33:26.361596000 -0500
# +++ /tmp/.DQ7X02	2025-01-22 06:33:26.361680000 -0500
# @@ -4,7 +4,7 @@
#  DejaVu Sans 20
#  Noto Sans CJK JP 20
#  Droid Sans Japanese 20
# -Cantarell 20
# +Cantarell 20 @wght=400
#  Amiri 20
#  DejaVu Sans Bold 20
#  Font Awesome 5 Free Solid Heavy 20
# 
not ok 4 /fontsets/mono

--- tests/fontsets/mono.orig	2025-01-18 22:41:03 UTC
+++ tests/fontsets/mono
@@ -4,7 +4,7 @@ Droid Sans Japanese 20
 DejaVu Sans 20
 Noto Sans CJK JP 20
 Droid Sans Japanese 20
-Cantarell 20
+Cantarell 20 @wght=400
 Amiri 20
 DejaVu Sans Bold 20
 Font Awesome 5 Free Solid Heavy 20
